matlab/distance.m

```diff
@@ -0,0 +1,18 @@
+function totalDistance =  distance(latA, lngA, latB, lngB)
+% Fully vectorized version of code
+
+    earthR = 6378137.0;
```

Comment:
use mean radius (~6371000) here. you are assuming a spherical approximation. welcome to the church of haversine formula by the way!

Reply:
Agreed and Corrected